### PR TITLE
Cleanup: Better ratio of commented classes

### LINKED ISSUE #13328: Cleanup: Better ratio of commented classes
Still many classes in the system had been left uncommented (without a class/traits comment).

```Smalltalk
stereotypes := OrderedCollection new.
stereotypes
	addAll: Smalltalk allTraits;
	addAll: Smalltalk allClasses.
	
stereotypes removeAll: TestCase allSubclasses.
stereotypes removeAll: BaselineOf allSubclasses.
violating := stereotypes select: [:each | each comment isEmpty ]
```

Using that script we could add a release test that checks that the size is not growing and over time fix more and more classes by commenting them

### PR DESCRIPTION
**Pull request by astares**
Fix #13328

### CODE REVIEW

**Review by jecisc — APPROVED** [review]
I like this idea :)

**Comment by jecisc** [comment]
Test failure unrelated

File: src/ReleaseTests/ReleaseTest.class.st [hunk 1]
```diff
@@ -410,6 +410,21 @@ ReleaseTest >> testRPackageOrganizer [
...
+	| stereotypes violating |
+	stereotypes := OrderedCollection new.
+	stereotypes
+		addAll: Smalltalk allTraits;
+		addAll: Smalltalk allClasses.
+
+	stereotypes removeAll: TestCase allSubclasses.
+	stereotypes removeAll: BaselineOf allSubclasses.
+	violating := stereotypes select: [ :each | each comment isEmpty ].
+	self assert: violating size <= 914
```
Comment (privat) [inline src/ReleaseTests/ReleaseTest.class.st:425]: Maybe split the method into two, one that computes the violating classes and one that asserts its size

**Review by privat — APPROVED** [review]
good idea.

File: src/ReleaseTests/ReleaseTest.class.st (hunk 1 above)
Comment (astares) [reply src/ReleaseTests/ReleaseTest.class.st:425]: Why? As long as there is no other method requiring this it is not needed and we can all the time add if needed.

**Comment by jecisc** [comment]
To avoid this scenario:
- You comment a class of the system, uncommented classes goes down to 913 and test still pass
- You do a commit with a new uncommented class and the number goes back to 914 and the test still pass but we have a new uncommented class 

We can either:
- Check that the uncommented classes are all part of the list of actual violations and then new uncommented classes would fail the test
- Or check that the uncommented classes are exactly a list, and then we would also have to remove the classes we comment form the list and we ensure we do not remove the comment later. This is better but might be a hassle.

Personally I like solution 1 but I would keep the code to generate the tests in a method comment so that we can regenerate the test when we comment a bunch of classes and we think about it.

**Comment by astares** [comment]
Lets keep it simple - we had/have similar release tests who just checked for size of violations. We then applied cleaning and step by step brough the counters down adopting the numbers.

The thing is that we keep improving and the test just assures that the quality level does not decrease....

**Pull request merged**